src/site/_includes/partials/course-app-bar.njk

```diff
@@ -10,5 +10,15 @@
       <a href="/learn/css/">Learn CSS!</a>
     </li>
   </ul>
-  <button class="w-ml--auto">Share</button>
+  <share-action class="gc-analytics-event app-bar__share w-ml--auto"
+    authors="{{ authors | expandAuthors(collections.authors) }}"
+    data-category="web.dev"
+    data-label="share"
+    data-action="click"
+    data-icon="share"
+    tabindex="0"
+    role="button"
```

Comment:
hm it's interesting that the existing share action is implemented this way. It seems like it would be a lot better for it to wrap a `<button>`

There's an existing `.w-button--svg` class that I created for when we want to inline svg. I wonder if we could do something like:

```
{% from 'macros/icon.njk' import icon with context %}
<share-action>
  <button class="w-button--svg">
    {{ icon('share') }}
  </button>
</share-action>
```

Reply:
oh i just realized w-button--svg is for buttons that only contain svg but this also contains text. let me fiddle with it a bit